### PR TITLE
pull request for the readme files in nodejs add content

### PR DESCRIPTION
**Pull request by getcoder-top**
the readme file in nodejs is updated

### CODE REVIEW

**Comment by getcoder-top** [comment]
please review,

**Comment by getcoder-top** [comment]
@kamranahmedse please review

**Comment by getcoder-top** [comment]
@kant @ryanwi @dhruvasagar @kamranahmedse please review

File: content/roadmaps/113-qa/content/readme.md [hunk 1]
```diff
@@ -1 +1,6 @@
-# 
\ No newline at end of file
+#
```
Comment (kamranahmedse) [inline content/roadmaps/113-qa/content/readme.md:1]: This will not have any impact

File: content/roadmaps/107-nodejs/content/readme.md [hunk 2]
```diff
@@ -1 +1,7 @@
-# 
\ No newline at end of file
+#
```
Comment (kamranahmedse) [inline content/roadmaps/107-nodejs/content/readme.md:1]: This will not have any impact, it's just the dummy file in roadmaps and will be removed

**Pull request closed without merging**